Doc: Prepare for 1.11.0-RC1 release (#8859)
OpenTTD/OpenTTD: changelog.txt

1.11.0-RC1 (2021-03-14)
------------------------------------------------------------------------

> Feature: Option to (dis-)allow accelerated video drivers (#8819)

Reviewer: "GPU accelerated" ?

Feature: Display refresh rate game option (#8813)
Feature: Allow custom width/height of screenshot and making heightmap screenshots via console (#8804)
Feature: Allow filtering on name in rail station window (#8706)

> Feature: Allow setting maximum zoom level at which sprites are drawn (#8604)

Reviewer: Tricky thing to summarize to a single line, but still feels a bit off to me
Something like "Add setting to define maximum sprite zoom level", maybe?

Author: Went with "Setting for highest resolution of sprites to use", as that is how the setting is called ingame :) This really has been debated way too much :D :D Funny, how difficult some things are to describe in a single sentence :)

> Add: Make NewGRF Scanner / World Generation update more smooth and make aborting them react faster (#8830)

Reviewer: "more smooth" -> "smoother"
"them" -> "it"

Add: Malaysia Ringgit as Currency (#8783)

> Add: Filter for "engines only" in build train window (#8733)

Reviewer: ```suggestion
Add: "Engines only" filter in build train window (#8733)
```

Change: De-limit framerate window's framerate (#8772)

> Change: Clarify what effect town interactions have (#8744)

Reviewer: Minor enough to remove?

Author: Nah, it is fine. We shouldn't make this too difficult for ourselves.

Change: Don't show global goals in company goal windows (#8709)
Change: Recolour graph windows to brown (#8700)

> Fix #8855: If bootstrap failed, it could end with an empty screen instead of error (#8856)

Reviewer: Reverse it a bit - 
```suggestion
Fix #8855: Bootstrap could result in an empty screen on error (#8856)
```

Fix #8851: Don't allow infinite "exec" depth in script, but limit to 10 deep (#8852)

> Fix #8647: Order of drawing tram catenaries wasn't correct (#8843)

Reviewer: "Incorrect drawing order of tram catenary sprites"

Fix #8711: Having gui_zoom lower than zoom_min causes crashes (#8835)
Fix #8810: "aircraft out of fuel" news shows the wrong place (#8832)
Fix #8833: Don't reload NewGRFs when we are shutting down (#8830)
Fix: Scale padding between elements the same as regular padding (#8829)
Fix #8808: [OSX, OpenGL] Crash on switching blitters due to double-mapping the video buffer (#8822)
Fix #8784: Using Alt+Enter didn't update the fullscreen toggle visibly (#8820)
Fix #8817: Keep NewGRF order for object class sorting (#8818)
Fix #8809: Crash when removing airport when hangar window open (#8815)
Fix #8799: Crash when Search Internet in Multiplayer (#8801)

> Fix #8775: [Win32] Don't create the main window when alt-tabbing back into fullscreen (#8792)

Reviewer: I'd suggest "Alt-Tabbing" for consistency with Alt+Enter above, but that feels a bit off. Perhaps "switching window" ?

Author: Pretty sure everyone (like really: everyone) understands Alt+Tabbing :)

Fix #8774: Black screenshots when using 40bpp-blitter (#8791)
Fix: [OSX] Hide dock when entering fullscreen (#8789)

> Fix: Bootstrap was only showing a black screen (#8788)

Reviewer: Tense is wrong and/or different to the others

Reviewer 2: I don't think many players know what bootstrap is in OpenTTD

1.11.0-RC1 (2021-03-14)
------------------------------------------------------------------------
Feature: Allow setting maximum zoom level at which sprites are drawn (#8604)
Feature: Allow filtering on name in rail station window (#8706)
Feature: Allow custom width/height of screenshot and making heightmap screenshots via console (#8804)

> Add: Make mNewGRF Scanner / World Generation update more smooth and make aborting them react faster (#8830)

Reviewer 3: Why do you have `mNewGRF Scanner` instead of just `NewGRF Scanner`?

Author: Because people make typos? No need to be mean about it :P

> Add: Display refresh rate game option (#8813)

Reviewer: Should be Feature

> Add: [Video] Move GameLoop into its own thread (#8741)

Reviewer: Is a codechange with (almost) no visible user impact, remove

> Add: Option to (dis-)allow accelerated video drivers (#8819)

Reviewer: Feature?

Add: Filter for "engines only" in build train window (#8733)

> Add: Malaysia Ringgit as Currency (#8783)

Reviewer: Wasn't this in beta2?

Author: Indonesia Rupiah was ;)

Change: Recolour graph windows to brown (#8700)
Change: Don't show global goals in company goal windows (#8709)
Change: Clarify what effect town interactions have (#8744)
Change: De-limit framerate window's framerate (#8772)
Fix #8855: If bootstrap failed, it could end with an empty screen instead of error (#8856)
Fix #8851: Don't allow infinite "exec" depth in script, but limit to 10 deep (#8852)
Fix #8647: Order of drawing tram catenaries wasn't correct (#8843)
Fix #8833: Don't reload NewGRFs when we are shutting down (#8830)
Fix #8711: Having gui_zoom lower than zoom_min causes crashes (#8835)
Fix #8810: "aircraft out of fuel" news shows the wrong place (#8832)

> Fix: Scale PIP-padding the same as regular padding (#8829)

Reviewer: PIP is internal, is there some more user-friendly term for it?

Fix #8817: Keep NewGRF order for object class sorting (#8818)

> Fix #8784: Using alt+enter didn't update the fullscreen toggle visibly (#8820)

Reviewer: Alt+Enter

> Fix: Broken company colours for 40bpp-blitter (#8821)

Reviewer 4: This is actually a fix for `Fix #8774: Black screenshots when using 40bpp-blitter (#8791)`, so doesn't really need to be in  the changelog.
